assets/src/edit-story/components/canvas/layout.js

```diff
@@ -171,7 +196,12 @@ function useLayoutParams(containerRef) {
     // space for the page is:
     const maxWidth = width;
     const maxHeight =
-      height - HEADER_HEIGHT - HEADER_SPACE - MENU_HEIGHT - CAROUSEL_HEIGHT;
+      height -
+      HEADER_HEIGHT -
+      HEADER_SPACE -
+      MENU_HEIGHT -
+      MENU_SPACE -
```

Comment:
On naming: The `MENU_SPACE` is basically the extra margin the menu needs? Reading it at first it felt like `HEADER_SPACE` would be the full space the header takes and `MENU_SPACE` would be the full space the menu takes. But `HEADER_HEIGHT` could also mean the full height -- so the "space" must actually mean the extra margin/room that's needed?

Could we perhaps use `*_MARGIN` or `*_PADDING` instead of `*_SPACE` or/and create constants for the full heights of these two and use those instead here (e.g. `HEADER_SPACE = HEADER_HEIGHT + HEADER_MARGIN` or sth like that)?

Reply:
Renamed to `_GAP`, good call. And the reason for the split is that they're not used together in the grid, so we need separate constants for it anyway.